soc/arm/nuvoton_npcx/common/scfg.c

```diff
@@ -129,6 +131,17 @@ void npcx_host_interface_sel(enum npcx_hif_type hif_type)
 	SET_FIELD(inst_scfg->DEVCNT, NPCX_DEVCNT_HIF_TYP_SEL_FIELD, hif_type);
 }
 
+void npcx_dbg_freeze_enable(bool enable)
+{
+	const uintptr_t dbg_base = npcx_scfg_cfg.base_dbg;
+
+	if (enable) {
+		NPCX_DBGFRZEN3(dbg_base) &= ~BIT(7);
```

Comment:
Could you add a `#define` for `7`? Looks like other bit fields have them.

Reply:
Done.